### ISSUE TITLE
[FR] Add class for draft and revision

### ISSUE THREAD

**Issue by jonleverrier** [lindseydiloreto/craft-cpbodyclasses#4]
Would you consider adding a `draft` and `revision` class like you did for `new` for entries here https://github.com/doublesecretagency/craft-cpbodyclasses/commit/9225cad361e8dc7f1a39a7dd9998d48f888d6044 ? I've got this working on local dev and find it very useful !

**Comment by lindseydiloreto** [comment]
Sure, I could probably work that in! What specific changes did you make to determine the `draft`/`revision` status?

**Comment by jonleverrier** [comment]
I used similar logic you applied for checking for new entries. I’ll send you something on Discord when I’m back at my machine :)

**Comment by jonleverrier** [comment]
Just to say @lindseydiloreto, I sent you a PM on Discord with a Gist.

**Comment by lindseydiloreto** [comment]
Hi @jonleverrier, I got your PM. I thought I had responded, sorry.

It's still on my list... I've been swamped with client work over the past few weeks, but I'll try to get to it as soon as possible! 🙂

**Comment by lindseydiloreto** [comment]
Hi @jonleverrier,

I've just added this feature, sorry for the notable delay. I actually did it a little differently from your gist. Go ahead and update to the latest version (`2.2.0`) when you get a chance.

You now have this checkbox available to you...

<img width="698" alt="Ω 2020-04-04 at 11 27 35 PM" src="https://user-images.githubusercontent.com/5309692/78468392-71a0f000-76cc-11ea-8974-e524f53140a9.png">

And when you are editing an entry, the body tag will now include _one_ of the following classes...

```
entryversion-new
entryversion-draft
entryversion-revision
entryversion-current
```

Hope that helps, let me know if you have any questions!
